tests/torch/dataloaders/test_data/adcf7d18-0510-35b0-a2fa-b4cea13a6d76/map/log_map_archive_adcf7d18-0510-35b0-a2fa-b4cea13a6d76____PIT_city_57819.json

```diff
@@ -0,0 +1 @@
}, {"x": 1350.0, "y": 133.71, "z": 14.28}, {"x": 1350.0, "y": 145.18, "z": 13.96}, {"x": 1359.81, "y": 140.76, "z": 14.21}, {"x": 1361.08, "y": 140.25, "z": 14.23}, {"x": 1362.06, "y": 139.98, "z": 14.23}, {"x": 1363.03, "y": 140.06, "z": 14.23}, {"x": 1363.82, "y": 140.35, "z": 14.23}, {"x": 1364.34, "y": 140.91, "z": 14.22}, {"x": 1365.3, "y": 142.88, "z": 14.19}, {"x": 1368.79, "y": 150.0, "z": 13.93}, {"x": 1380.87, "y": 150.0, "z": 13.95}, {"x": 1378.85, "y": 146.27, "z": 14.08}, {"x": 1374.85, "y": 137.97, "z": 14.3}, {"x": 1374.6, "y": 137.14, "z": 14.31}, {"x": 1374.4, "y": 136.22, "z": 14.31}, {"x": 1374.49, "y": 135.45, "z": 14.32}, {"x": 1374.77, "y": 134.69, "z": 14.34}, {"x": 1375.32, "y": 133.93, "z": 14.38}, {"x": 1376.19, "y": 133.1, "z": 14.44}, {"x": 1377.39, "y": 132.42, "z": 14.48}, {"x": 1391.34, "y": 125.83, "z": 14.8}, {"x": 1400.8, "y": 121.3, "z": 15.0}, {"x": 1412.93, "y": 115.5, "z": 15.25}, {"x": 1421.65, "y": 111.39, "z": 15.35}, {"x": 1427.74, "y": 108.48, "z": 15.44}, {"x": 1437.6, "y": 103.68, "z": 15.52}, {"x": 1443.4, "y": 101.12, "z": 15.57}, {"x": 1444.96, "y": 100.57, "z": 15.61}, {"x": 1446.02, "y": 100.29, "z": 15.61}, {"x": 1446.65, "y": 100.16, "z": 15.62}, {"x": 1447.33, "y": 100.1, "z": 15.62}, {"x": 1447.87, "y": 100.14, "z": 15.62}, {"x": 1448.4, "y": 100.16, "z": 15.62}, {"x": 1448.75, "y": 100.16, "z": 15.62}, {"x": 1449.27, "y": 100.35, "z": 15.62}, {"x": 1449.71, "y": 100.47, "z": 15.62}, {"x": 1450.16, "y": 100.65, "z": 15.61}, {"x": 1450.44, "y": 100.83, "z": 15.61}, {"x": 1450.77, "y": 101.09, "z": 15.61}, {"x": 1451.04, "y": 101.38, "z": 15.61}, {"x": 1451.32, "y": 101.73, "z": 15.6}, {"x": 1451.55, "y": 102.18, "z": 15.6}, {"x": 1460.69, "y": 120.0, "z": 15.1}, {"x": 1470.79, "y": 120.0, "z": 14.92}, {"x": 1463.17, "y": 104.98, "z": 15.31}, {"x": 1462.3, "y": 103.42, "z": 15.41}, {"x": 1461.97, "y": 102.67, "z": 15.43}, {"x": 1461.83, "y": 102.3, "z": 15.46}, {"x": 1461.7, "y": 101.83, "z": 15.48}, {"x": 1461.68, "y": 101.34, "z": 15.48}, {"x": 1461.68, "y": 100.86, "z": 15.49}, {"x": 1461.72, "y": 100.47, "z": 15.5}, {"x": 1461.85, "y": 100.02, "z": 15.5}, {"x": 1462.01, "y": 99.69, "z": 15.52}, {"x": 1462.3, "y": 99.28, "z": 15.58}, {"x": 1462.69, "y": 98.92, "z": 15.65}, {"x": 1463.18, "y": 98.53, "z": 15.69}, {"x": 1463.68, "y": 98.24, "z": 15.74}, {"x": 1465.22, "y": 97.41, "z": 15.82}, {"x": 1486.35, "y": 87.37, "z": 16.89}, {"x": 1505.05, "y": 78.47, "z": 18.04}, {"x": 1514.38, "y": 73.95, "z": 18.49}, {"x": 1515.31, "y": 73.45, "z": 18.52}, {"x": 1516.22, "y": 73.05, "z": 18.51}, {"x": 1517.0, "y": 72.73, "z": 18.61}, {"x": 1517.84, "y": 72.43, "z": 18.62}, {"x": 1518.43, "y": 72.33, "z": 18.62}, {"x": 1519.1, "y": 72.3, "z": 18.63}, {"x": 1519.75, "y": 72.47, "z": 18.64}, {"x": 1520.65, "y": 72.94, "z": 18.66}, {"x": 1521.35, "y": 73.5, "z": 18.68}, {"x": 1522.07, "y": 74.41, "z": 18.71}, {"x": 1531.69, "y": 93.82, "z": 18.13}, {"x": 1544.61, "y": 120.0, "z": 17.14}], "id": 1413627}}}
```

Comment:
Repo size is still mostly reasonable now, but I wonder if we should migrate to something LFS-based in the future to prevent bloat.